bin/isympy

```diff
@@ -1,357 +1,5 @@
-    but with this option enabled, any undefined name will be
-    automatically created as a Symbol.
-
-    Note that this is intended only for interactive, calculator style
-    usage. In a script that uses SymPy, Symbols should be instantiated
-    at the top, so that it's clear what they are.
-
-    This will not override any names that are already defined, which
-    includes the single character letters represented by the mnemonic
-    QCOSINE (see the "Gotchas and Pitfalls" document in the
-    documentation). You can delete existing names by executing "del
-    name".  If a name is defined, typing "'name' in dir()" will return True.
-
-    The Symbols that are created using this have default assumptions.
-    If you want to place assumptions on symbols, you should create them
-    using symbols() or var().
-
-    Finally, this only works in the top level namespace. So, for
-    example, if you define a function in isympy with an undefined
-    Symbol, it will not work.
-
-    See also the -i and -I options.
-
--i, --int-to-Integer (requires at least IPython 0.11)
-
-    Automatically wrap int literals with Integer.  This makes it so that
-    things like 1/2 will come out as Rational(1, 2), rather than 0.5.  This
-    works by preprocessing the source and wrapping all int literals with
-    Integer.  Note that this will not change the behavior of int literals
-    assigned to variables, and it also won't change the behavior of functions
-    that return int literals.
-
-    If you want an int, you can wrap the literal in int(), e.g. int(3)/int(2)
-    gives 1.5 (with division imported from __future__).
-
--I, --interactive (requires at least IPython 0.11)
-
-    This is equivalent to --auto-symbols --int-to-Integer.  Future options
-    designed for ease of interactive use may be added to this.
-
--D, --debug
-
-    Enable debugging output.  This is the same as setting the
-    environment variable SYMPY_DEBUG to 'True'.  The debug status is set
-    in the variable SYMPY_DEBUG within isympy.
-
--- IPython options
-
-    Additionally you can pass command line options directly to the IPython
-    interpreter (the standard Python shell is not supported).  However you
-    need to add the '--' separator between two types of options, e.g the
-    startup banner option and the colors option. You need to enter the
-    options as required by the version of IPython that you are using, too:
-
-    in IPython 0.11,
-
-        $isympy -q -- --colors=NoColor
-
-    or older versions of IPython,
-
-        $isympy -q -- -colors NoColor
-
-See also isympy --help.
-
-"""
-
-import os
-import sys
-
-# hook in-tree SymPy into Python path, if possible
-
-isympy_path = os.path.abspath(__file__)
-isympy_dir = os.path.dirname(isympy_path)
-sympy_top = os.path.split(isympy_dir)[0]
-sympy_dir = os.path.join(sympy_top, 'sympy')
-
-if os.path.isdir(sympy_dir):
-    sys.path.insert(0, sympy_top)
-
-# DO NOT IMPORT SYMPY HERE!
```

Comment:
This is the original reason for this comment https://github.com/sympy/sympy/issues/7048. Let's double check that the `-t python` flag works.

Reply:
I just checked and it doesn't work (the ground types are set to `gmpy`). I think it is not possible to make this work with `python -m sympy`, because Python already imports sympy first when that is run. So we might have to do some hacks to make the -t flag work.